### ISSUE TITLE
ExoPlayer imports migration

### ISSUE THREAD

**Issue by abouda** [androidx/media#601]
My intention here is to be able to use the latest ExoPlayer version. So I am trying to migrate the ExoPlayer imports to media3 imports.
The issue is that `MediaSessionConnector` does not exist in media3. I understand that this class will no longer be needed in the future and is replaced by media3 MediaSession. But I cannot migrate now because of some regressions in the media3 MediaSession.

The question is, is it not possible to only migrate ExoPlayer to keep using the future ExoPlayer versions?

**Comment by marcbaechinger** [comment]
No, this is not possible. Once ExoPlayer is migrated to `androidx.media3` the `MediaSessionConnector` can't be used anymore, because it only accepts a player with the old package.
